.github/workflows/test-queries.yml

```diff
@@ -59,10 +59,12 @@ jobs:
       - uses: ilammy/msvc-dev-cmd@v1
       - uses: actions/setup-node@v3
 
+      - name: Install tree-sitter CLI
+        run: npm i -g tree-sitter-cli@0.20.7
```

Comment:
It might not be bad to be just on the latest . I think the version for the old CI was only there to work around that we would only download on Ubuntu.

Reply:
Sure, I can fix this

Reply:
Done